lib/auth.js

```diff
@@ -269,11 +269,29 @@ internals.Auth.prototype._authenticate = function (request, next) {
 
         // Check scope
 
+        var expandScope = function (scope) {
+
+            return scope.replace(/{([^{}]+)}/g, function (_, context) {
+
+                return Hoek.reach({
+                    params: request.params,
+                    query: request.query,
+                    payload: request.payload
+                }, context);
+            });
+        };
+
         if (config.scope) {
+            if (typeof config.scope === 'string') {
+                config.scope = [config.scope];
```

Comment:
if by multiple bits, you mean would they be allowed to compose their scope with multiple dynamic values, then yes definitely.